### DISCUSSION TITLE
Message -- nvcc version installed not supported by pytorch!!

### DISCUSSION THREAD

**Discussion by jjherlan** [cnr-isti-vclab/TagLab#111] [Q&A]
Hello, 

I receive the following message when I try to install the dependencies for TagLab into my virtual environment (TagLab folder is located in the virtual env named **py39_venv**). Would you happen to know why PyTorch and CUDA 11.8 are not communicating? Thank you in advance.

I am using Python 3.9.X - but the same thing occurred with Python 3.10.X.

- Ubuntu 22.04 LTS
- Python 3.9.17 

`python3 install.py`

Found NVCC version: 11.8

nvcc version installed not supported by pytorch!!
Something is wrong with NVCC. Do you want to install the CPU version of pythorch? [Y/n]n

Traceback (most recent call last):
  File "/home/jherlan/py39_venv/TagLab/install.py", line 115, in <module>
    raise Exception('Installation aborted. Install a proper NVCC version or set the pythorch CPU version.')
Exception: Installation aborted. Install a proper NVCC version or set the pythorch CPU version.

**Comment by maxcorsini** [comment]
Hi jjherlan,

The message is not correct. CUDA 11.8 is now supported by Pytorch but it is not supported by TagLab installer. We will fix this ASAP, in the meanwhile you can downgrade the version of CUDA to 11.7 .

Sorry for the inconvenience, the versions of CUDA supported by Pytorch change continuously.

**Reply by jjherlan** [reply]
Thanks, as always, for the reply and the support.
